packages/vat-data/src/baggage-place.js

```diff
@@ -0,0 +1,71 @@
+/** @typedef {import('./types').Baggage} Baggage */
+/** @typedef {import('@agoric/store/exported').Place} Place */
+
+import {
+  defineVirtualExoClass,
+  defineVirtualExoClassKit,
+  prepareExo,
+  prepareExoClass,
+  prepareExoClassKit,
+} from './exo-utils.js';
+import {
+  makeScalarBigMapStore,
+  makeScalarBigSetStore,
+  makeScalarBigWeakMapStore,
+  makeScalarBigWeakSetStore,
+  provideDurableMapStore,
+  provideDurableSetStore,
+  provideDurableWeakMapStore,
+  provideDurableWeakSetStore,
+} from './vat-data-bindings.js';
+
+const { Fail } = assert;
+
+/**
+ * @type {Place}
+ */
+export const virtualPlace = harden({
+  exo: () => Fail`Virtual singleton exos are not needed.`,
+  exoClass: defineVirtualExoClass,
+  exoClassKit: defineVirtualExoClassKit,
+
+  mapStore: (label, options = {}) =>
+    makeScalarBigMapStore(label, { durable: false, ...options }),
```

Comment:
Since `false` is `durable`'s default, putting it at the beginning is a noop. Should it go at the end, overriding an explicit setting? If so, we should change the `durable: true` in `provide...Store` to go at the end too.

Or should we simply omit it here, more simply preserving the current meaning, where it could be overridden?